src/Compute/Compute/help/Invoke-AzVMRunCommand.md

```diff
@@ -260,4 +260,9 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## NOTES
 
+This command leaves a copy of every script that it runs on the Linux VM it runs against each time it is invoked.
```

Comment:
Script belonging to only the latest invocation is retained. The previous scripts are actually deleted. It was security Ask.